cargo-espflash/src/main.rs

```diff
@@ -252,9 +263,17 @@ fn save_image(
     metadata: CargoEspFlashMeta,
     cargo_config: CargoConfig,
 ) -> Result<()> {
-    let target = cargo_config
-        .target()
-        .ok_or_else(|| NoTargetError::new(None))?;
+    let target = matches
+        .build_args
+        .target
+        .as_deref()
+        .or_else(|| cargo_config.target());
```

Comment:
why not use the `ok_or_else` here as before, instead of the `is_none` and `unwrap` there is now

Reply:
Of course, thanks for pointing that out.